.travis.yml

```diff
@@ -55,7 +55,7 @@ jobs:
     - stage:
       <<: *tox
       addons:
-        firefox: latest-nightly
+        firefox: latest-dev
```

Comment:
I think this is fine as a temporary fix. I would like to see `latest-nightly` commented out with a link to a bug for re-enabling it. We should unblock master then investigate why it's failing on Nightly and try to get back on Nightly.